src/box/memtx_tx.c

```diff
@@ -955,6 +955,7 @@ memtx_tx_story_delete(struct memtx_story *story)
 	for (uint32_t i = 0; i < story->index_count; i++) {
 		assert(story->link[i].newer_story == NULL);
 		assert(story->link[i].older_story == NULL);
+		assert(rlist_empty(&story->link[i].read_gaps));
```

Comment:
Nit: would be nice to describe these expectations in the function description.

Reply:
Fixed, moved the comment above to the function level. I guess _fully unlinked_ also describes gaps.